deepchem/feat/dft_featurizer.py

```diff
@@ -0,0 +1,332 @@
+from __future__ import annotations
+import os
+from abc import abstractmethod, abstractproperty
+from typing import List, Dict, Optional, Union
+import numpy as np
+import torch
+import dqc
+from dqc.system.mol import Mol
+from dqc.system.base_system import BaseSystem
+from dqc.grid.base_grid import BaseGrid
+from deepchem.utils.dftutils import KSCalc
+
+class System(dict):
+    """
+    Interface to the system in the dataset.
```

Comment:
This docstring is a bit confusing; as a user of DeepChem, what should I use this class for? Remember that a user doesn't have much context for this code so the docstring should provide that context

Reply:
As another note, these classes should be added to the docs